static/css/v3/boostlook-v3.css

```
@@ -0,0 +1,5583 @@
+.boostlook .doc .listingblock:has(.source-toolbox) .title:not(:has(a, span, p, code, pre)),
+.boostlook .listingblock:has(.source-toolbox) .title:not(:has(a, span, p, code, pre)) {
+  position: absolute;
+  top: 1px;
+  height: 2rem;
+  max-width: calc(100% - 5rem);
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: 0;
+  padding-left: var(--spacing-size-sm);
+  display: flex;
```

Stylelint (17.14.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)